src/factory.js

```diff
@@ -173,6 +189,19 @@ export class Factory {
     return tuplet;
   }
 
+  Beam(params) {
+    params = setDefaults(params, {
+      notes: [],
+      options: {
+        autoStem: false,
+      },
+    });
+
+    const beam = new Beam(params.notes, params.options.autoStem).setContext(this.context);
```

Comment:
Yes, that is fine. This is actually a good case for it, and a way to break out of long unnamed parameter lists.